lib/cc/cli/path_fingerprint.rb

```diff
@@ -0,0 +1,33 @@
+require "digest/md5"
+
+module CC
+  module CLI
+    class PathFingerprint
+      def initialize(output)
+        @output = output
+      end
+
+      def compute
+        md5 = Digest::MD5.new
+        md5 << key
+        md5.hexdigest
+      end
+
+      def key
+        "#{path}|#{check_name}"
```

Comment:
Could you explain the benefit of adding the separators separately? I don't think they're even necessary, but it makes for a more readable key.